src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -8,6 +8,7 @@
 using System.Globalization;
 using System.IO;
 using System.Linq;
+using System.Net.NetworkInformation;
```

Comment:
Is this required?

Reply:
No, VS sometimes adds imports automatically for typos.